core/sdk-core/src/main/java/software/amazon/awssdk/core/interceptor/SdkInternalExecutionAttribute.java

```diff
@@ -30,6 +32,13 @@
      */
     public static final ExecutionAttribute<Boolean> IS_FULL_DUPLEX = new ExecutionAttribute<>("IsFullDuplex");
 
+    /**
+     * The key to store the {@link CompletableFuture} returned by {@link AsyncResponseTransformer#prepare()} method
+     * in the first attempt of a request. This is used only for async streaming requests
+     */
+    public static final ExecutionAttribute<CompletableFuture<?>> STREAMING_ASYNC_RESPONSE_TRANSFORMER_FUTURE =
```

Comment:
`AsyncResponseTransformer` is only used for streaming operations, should we drop the prefix `STREAMING_`?

Reply:
Sure, will remove